pom.xml

```diff
@@ -13,15 +13,14 @@
   <artifactId>gson-parent</artifactId>
   <version>2.9.1-SNAPSHOT</version>
   <packaging>pom</packaging>
-
+ 
```

Comment:
This doesn't seem necessary?

Reply:
Yes sorry, i was trying to solve the CLA problem, this is actually my first ever PR and i thought by changing my email in the next commit would solve this problem